### DISCUSSION TITLE
Welcome to cesium-extends Discussions!

### DISCUSSION THREAD

**Discussion by hongfaqiu** [hongfaqiu/cesium-extends#9] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by WangJie-7** [comment]
作者您好！这个项目的api文档里面的geojson-render模块的案例不知是否能补充清楚，针对于六种类型可视化，如果可以我将不胜感激！

**Reply by hongfaqiu** [reply]
下周可以安排

**Reply by shyser** [reply]
geojson-render这个文档页面还有个问题，由于渲染了好几个实例，导致每次打开这个页面时都会卡上半天。我是把这个页面导出为PDF来查看的。如果可以做到的话，建议默认不要渲染实例，分别提供一个开关按钮即可，需要看哪个的效果就自己点开哪个。
